tests/HomeworkCalculatorTesting.spec.js

```diff
@@ -0,0 +1,82 @@
+const { test, expect } = require('@playwright/test');
+/*const { BasicCalculatorPage } = require('../pages-Calculator/basicCalculatorPage');
+
+
+test.describe('', () => {
+    let page;
+    
+    test.beforeAll(async ({ browser }) => {
+      page = await browser.newPage();
+      startPage = new BasicCalculatorPage(page);
+
+        });
+   test.beforeEach(async () => {
+      await startPage.goto();
+    });
+  });*/
+       
+ 
+test('Testing page to be opened', async ({ page }) => {
+  await page.goto('https://testsheepnz.github.io/BasicCalculator');
+  const calculatorPageTop = await page.isVisible('#page-top');
+  expect(calculatorPageTop).toBe(true);
+  
+});
+
+test('Check that calculator is working', async ({ page }) => {
+  await page.goto('https://testsheepnz.github.io/BasicCalculator');
+  await page.waitForSelector('#page-top');
+  await page.fill('#number1Field', '5');
+  await page.fill('#number2Field', '3');
+  await page.click('#calculateButton');
+  const result2Content = await page.isVisible('#numberAnswerField');
```

Comment:
isVisible returns boolean (true/false). So the following assertion will always fail.